### PR TITLE
WIP: Feature/update documentation

### PR DESCRIPTION
**Pull request by anemonekk**
## Reason for this PR:
 - The documentation provided in the README.md file needed to be improved to ease the use and further development of our repository.

## Changes in this PR:
 - Change sections in README.md
 - Add more explanatory conceptual parts
 - Add a folder structure
 - Add table of contents
 - Update old legacy descriptions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +236,16 @@ Get shell on CLI container `kubectl exec -n hlf-production-network {CLI-POD} -it
 
 Get logs of container `kubectl logs {POD} -n hlf-production-network`.
 
-You can omit the namespace parameter if you set the context of kubectl `kubectl config set-context --current --namespace=hlf-production-network`.
+You can omit the namespace parameter, if you set the context of kubectl `kubectl config set-context --current --namespace=hlf-production-network`.
+
+### Debugging
+
+For debugging, we provide a few scripts in the folder scripts/debug. When executing ```podShell.sh```, input the pod name in order to run a shell on the specific pod. For viewing the logs of a specific pod, execute ```getLogs.sh``` with the respective pod name.
```
Comment (palkra) [inline README.md:243]: ./podShell.sh deployment-name [container name]
./getLogs.sh deployment-name [containername]

File: README.md [hunk 2]
```diff
@@ -4,33 +4,229 @@
...
+\
+During our development, the following references were particularly useful for us: <!---TODO: check and order-->  
+* https://hyperledger-fabric.readthedocs.io/en/release-2.1/deployment_guide_overview.html
+* https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html
+* (https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/cadeploy.html) 
+* [channel configuration](https://hyperledger-fabric.readthedocs.io/en/release-1.4/configtx.html?channel-configuration-configtx)
+* [chaincode lifecycle](https://hyperledger-fabric.readthedocs.io/en/release-2.0/chaincode_lifecycle.html)
+<!---* (https://developer.ibm.com/technologies/blockchain/tutorials/hyperledger-fabric-kubernetes-cluster-tls-rhel/#b-create-hyperledger-fabric-manifests-for-kubernetes)
```
Comment (palkra) [inline README.md:91]: Remove links since urls not valid anymore

File: README.md [hunk 3]
```diff
@@ -4,33 +4,229 @@
...
+To reset the network, execute `./deleteNetwork.sh`. You can also delete everything and restart the network directly using `./restartNetwork.sh`. The command `minikube stop` stops minikube.
+
 
 ## Network
 
-The initial topology of the operations guide implements the most interesting use cases of hyperledger fabric besides multiple orderers.
-![Network Topology](https://hyperledger-fabric-ca.readthedocs.io/en/latest/_images/network_topology.png)
-There are three organizations, one providing the orderer service and two hosting two peers each on a shared channel. We deploy an external TLS CA which provides TLS certificates for all containers.
+The initial network topology suggested by the [operations guide](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html) implements the most interesting use cases of hyperledger fabric besides multiple orderers.
+<!---TODO: multiple orderers?-->
```
Comment (palkra) [inline README.md:25]: Remove bc we only have one orderer rn. Or write that we should use raft for several orderers since kafka is deprecated.

**Review by palkra — APPROVED** [review]

**Review by mco33415 — APPROVED** [review]

**Review by nklsd — APPROVED** [review]
looks good

**Pull request merged**